docs/_includes/backers.md

```diff
@@ -1,7 +1,8 @@
 ## Backers
 
-Find Mocha helpful?  Become a [backer](https://opencollective.com/mochajs#support) and support Mocha with a monthly donation.
+Find Mocha helpful? Become a [backer](https://opencollective.com/mochajs#support) and support Mocha with a monthly donation.
 
 <!-- markdownlint-disable MD034 -->
-{% for i in (0..29) %}[![](https://opencollective.com/mochajs/backer/{{ i }}/avatar.jpg)](https://opencollective.com/mochajs/backer/{{ i }}/website){: target="_blank" rel="noopener"}{% endfor %}
-{: .image-list id="_backers" }
+
+{% for i in (0..29) %}[![](https://opencollective.com/mochajs/backer/{{ i }}/avatar.jpg)](https://opencollective.com/mochajs/backer/{{ i }}/website){: target="\_blank" rel="noopener"}{% endfor %}
```

Comment:
What's the purpose of the added backslash?

Reply:
this could be prettier choking on the template code.

Reply:
all 11ty template stuff needs wrapping in 

```html
<!-- prettier-ignore -->

stuff

<!-- prettier-ignore-end -->
```